{"name": "test_add_error[test01]", "status": "passed", "parameters": [{"name": "a", "value": "'æ–‡'"}, {"name": "b", "value": "0.3"}, {"name": "expect", "value": "'TypeError'"}], "start": 1648449883396, "stop": 1648449883397, "uuid": "83492eb5-0731-475a-8935-7a660c22efd7", "historyId": "e8365f27c77897acfbd0b0e5d97f78ce", "testCaseId": "a3bfe621045782c9ea9a5de99829cd04", "fullName": "test_pytest.test_cal.TestCal#test_add_error", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('def_fixture')"}, {"name": "parentSuite", "value": "test_pytest"}, {"name": "suite", "value": "test_cal"}, {"name": "subSuite", "value": "TestCal"}, {"name": "host", "value": "IT-L-YANGYL"}, {"name": "thread", "value": "7092-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_pytest.test_cal"}]}